base/applications/network/ipconfig/lang/ro-RO.rc

```diff
@@ -82,14 +82,14 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_DNSNORECORD         "\tNo records of type %ls\n\n"
-    IDS_DNSNONAME           "\tName does not exist.\n\n"
-    IDS_DNSFLUSHERROR       "Could not flush the DNS Resolver Cache: "
-    IDS_DNSFLUSHSUCCESS     "Successfully flushed the DNS Resolver Cache.\n"
-    IDS_DHCPNOTCONNECTED    "No operation can be performed on %ls while it has its media disconnected.\n"
-    IDS_DHCPNOTENABLED      "Adapter %ls is not enabled for DHCP.\n"
-    IDS_DHCPNOADAPTER       "The operation failed as no adapter is in the state permissible for \nthis operation.\n"
-    IDS_DHCPRELEASED        "IP Address for adapter %ls has already been released.\n"
+    IDS_DNSNORECORD         "\tNu sunt înregistrări de tipul %ls\n\n"
+    IDS_DNSNONAME           "\tNumele nu există.\n\n"
+    IDS_DNSFLUSHERROR       "Nu a putut fi golită memoria de depozitare a rezolvatorului DNS: "
+    IDS_DNSFLUSHSUCCESS     "Memoria de depozitare a rezolvatorului DNS a fost golită cu succes.\n"
+    IDS_DHCPNOTCONNECTED    "Nu poate fi efectuată nicio operațiune %ls atât timp cât media este deconectată.\n"
```

Comment:
In Romanian, "media" (with its inarticulate form "medie"), both from what [DEX says](https://dexonline.ro/definitie/medie/definitii) and as far as I remember from real life, it only refers to the arithmetic term "average". I assume you didn't intended to overload the meaning of this word in Romanian, and wanted instead to say "mediul" (with its inarticulate form "mediu"), which is already used in ReactOS translations?